lib/rls/api.rb

```diff
@@ -33,6 +33,36 @@ def player(id, platform = Platform::Steam)
       Player.new(response)
     end
 
+    # Retrieve a batch of players
+    # @example Retrieve a batch of players from multiple platforms
+    #   client.players(
+    #     76561198033338223, 1,
+    #     76561197981122126, RLS::Platform::Steam,
+    #     'Wizwonk', 2,
+    #     'Loubleezy', RLS::Platform::XboxOne
+    #   )
+    # @param *request_data [Array<String,Integer,#id>] list of player IDs and their platform
+    # @return [Array<Player>] Array of Player objects
```

Comment:
The exceptions this method raises needs to be documented here. You can see examples from the other methods I doc'd from my last PR.